lib/config/presets/bitbucket-server/index.spec.ts

```diff
@@ -35,6 +35,24 @@ describe('config/presets/bitbucket-server/index', () => {
       expect(res).toEqual({ from: 'api' });
     });
 
+    it('returns JSON5', async () => {
+      httpMock
+        .scope(bitbucketApiHost)
+        .get(`${basePath}/some-filename.json5`)
+        .query({ limit: 20000 })
+        .reply(200, {
+          isLastPage: true,
+          lines: [{ text: '{"from":"api"' }, { text: '}' }],
```

Comment:
you should use some JSON5 syntax in those tests. 😉

Reply:
> you should use some JSON5 syntax in those tests. 😉

good point! Thanks